src/lib/content.js

```diff
@@ -122,20 +122,38 @@ function loadProfiles(profile, list, csrf, hidesHistory, hidesAccountId) {
-      <input type="hidden" name="action" value="switchFromBasis">
-      <input type="hidden" name="src" value="nav">
-      <input type="hidden" name="roleName" value="${item.role_name}">
-      <input type="hidden" name="account" value="${item.aws_account_id}">
-      <input type="hidden" name="mfaNeeded" value="0">
-      <input type="hidden" name="color" value="${color}">
-      <input type="hidden" name="csrf" value="${csrf}">
-      <input type="hidden" name="redirect_uri" value="${redirectUri}">
-      <label for="awsc-recent-role-switch-0" class="awsc-role-color" style="background-color: #${color};">&nbsp;</label>
-      <input type="submit" class="awsc-role-submit awsc-role-display-name" name="displayName" value="${name}"
-            title="${item.role_name}@${item.aws_account_id}" style="white-space:pre"></form>
-    </li>`);
+    if (!item.image) {
+        list.insertAdjacentHTML('beforeend', Sanitizer.escapeHTML`<li>
+         <form action="https://signin.aws.amazon.com/switchrole" method="POST" target="_top" data-aesr-profile="${item.profile}">
+          <input type="hidden" name="action" value="switchFromBasis">
+          <input type="hidden" name="src" value="nav">
+          <input type="hidden" name="roleName" value="${item.role_name}">
+          <input type="hidden" name="account" value="${item.aws_account_id}">
+          <input type="hidden" name="mfaNeeded" value="0">
+          <input type="hidden" name="color" value="${color}">
+          <input type="hidden" name="csrf" value="${csrf}">
+          <input type="hidden" name="redirect_uri" value="${redirectUri}">
+          <label for="awsc-recent-role-switch-0" class="awsc-role-color" style="background-color: #${color};">&nbsp;</label>
+          <input type="submit" class="awsc-role-submit awsc-role-display-name" name="displayName" value="${name}"
+                title="${item.role_name}@${item.aws_account_id}" style="white-space:pre"></form>
+        </li>`);
+    } else {
+        list.insertAdjacentHTML('beforeend', Sanitizer.escapeHTML`<li>
+         <form action="https://signin.aws.amazon.com/switchrole" method="POST" target="_top" data-aesr-profile="${item.profile}">
+          <input type="hidden" name="action" value="switchFromBasis">
+          <input type="hidden" name="src" value="nav">
+          <input type="hidden" name="roleName" value="${item.role_name}">
+          <input type="hidden" name="account" value="${item.aws_account_id}">
+          <input type="hidden" name="mfaNeeded" value="0">
+          <input type="hidden" name="color" value="${color}">
+          <input type="hidden" name="csrf" value="${csrf}">
+          <input type="hidden" name="redirect_uri" value="${redirectUri}">
+          <label for="awsc-recent-role-switch-0" class="awsc-role-color"><img src=${item.image.replace(/"/g, '')} width="14em" height="14em"></label>
```

Comment:
The width and height attributes of img support only **px** as unit, so add the style attribute like `style="width: 1em; height: 1em"` to it.